packages/ui-kit/src/components/DataGrid/Loader.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames'
```

Comment:
Could you please also move this file into a separate folder `/packages/ui-kit/src/components/DataGrid/Loader/`?